src/layouts/DefaultLayout.vue

```diff
@@ -50,6 +50,19 @@
             </v-list-item>
           </v-list>
         </v-menu>
+        <v-list density="compact" v-if="navigationMenuComponents.length === 1">
+          <v-list-item
+            :value="navigationMenuComponents[0]"
+            :to="navigationMenuComponents[0].to"
```

Comment:
This errors as this is now immediately created on page load for a dynamic route that has not been setup yet: 
![image](https://github.com/Deltares/fews-web-oc/assets/144685504/13d7e15f-4187-4721-83da-1d213b0ea465)

As such we need to check if the route exists:
```vue
<v-list-item
    v-if="router.hasRoute(navigationMenuComponents[0].to.name)"
```